CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 [Releases](https://github.com/NetApp/harvest/releases)
 
+## 22.02.0 / 
+
+**IMPORTANT** Admin certificate through command ```bin/harvest admin tls create server``` will now be generated inside harvest ```cert``` folder instead of harvest root.
```

Comment:
**IMPORTANT** 🚧 Admin node certificate file location changed. Certificate files have been consolidated into the `cert` directory. You need to move your `admin-cert.pem` and `admin-key.pem` files to the `cert` directory.